### PR TITLE
Plugin put in

### PR DESCRIPTION
**Pull request by ryestew**
update to set location with local storage and a fix if no local storage is set for the plugin

### CODE REVIEW

**Comment by ryestew** [comment]
wait to merge this until I fix the issue https://github.com/ethereum/remix-ide/issues/1934

File: src/app/components/local-plugin.js [hunk 1]
```diff
@@ -34,11 +34,13 @@ module.exports = class LocalPlugin {
...
       methods: [],
+      location: 'swapPanel',
+      ...this.profile,
       hash: `local-${this.profile.name}`
     }
+    profile.events = profile.events || []
     profile.events = profile.events.filter((item) => { return item !== '' })
```
Comment (GrandSchtroumpf) [inline src/app/components/local-plugin.js]: You can do : 
```javascript
profile.events = (profile.events || []).filter(item => item !== '')
```

File: src/app/components/local-plugin.js [hunk 2]
```diff
@@ -120,6 +122,20 @@ module.exports = class LocalPlugin {
         return yo`<input class="form-control" onchange="${e => this.updateEvents(e, i)}" value="${event}" />`
       })}</div>`
     }
+    const radioLocations = (label, displayN) => {
+      const radioButton = (this.profile.location === label)
```
Comment (GrandSchtroumpf) [inline src/app/components/local-plugin.js]: You can just do : 
```javascript
return (this.profile.location === label) ? ... : ...
```

**Comment by ryestew** [comment]
Check if plugin name is not used yet #1934

**Pull request merged**